lib/linguist/languages.yml

```diff
@@ -4786,6 +4747,44 @@ reStructuredText:
   codemirror_mode: rst
   codemirror_mime_type: text/x-rst
   language_id: 419
+roff:
```

Comment:
Can't tell if you're trolling me now @Alhadis 😁 

I thought `roff` was the consensus here: https://github.com/github/linguist/issues/2382#issuecomment-196793686